src/cleanvision/issue_managers/duplicate_issue_manager.py

```diff
@@ -173,15 +173,24 @@ def _update_summary(self) -> None:
         self.summary = self.summary.astype({"issue_type": str})
 
     def _update_issues(self) -> None:
+        score_df = pd.DataFrame(index=self.issues.index)
+        score_df[get_score_colname(IssueType.EXACT_DUPLICATES.value)] = np.ones(
+            len(score_df)
+        )
+        score_df[get_score_colname(IssueType.NEAR_DUPLICATES.value)] = np.ones(
+            len(score_df)
+        )
+
         for issue_type in self.issue_types:
-            duplicated_images = []
+            score_col = get_score_colname(issue_type)
             for s in self.info[issue_type][SETS]:
-                duplicated_images.extend(s)
-            self.issues[
-                get_is_issue_colname(issue_type)
-            ] = self.issues.index.to_series().apply(
-                lambda x: True if x in duplicated_images else False
-            )
+                score = 1.0 / len(s)
+                score_df.loc[s, score_col] = score
```

Comment:
can you cover this line in unit test, if easy?

Reply:
Covered in unit test